pages/housing-assistance/index.md

```diff
@@ -35,6 +35,8 @@ social:
         links:
         - url: https://iris.custhelp.va.gov/app/ask
           label: "Ask a question online"
+        - url: https://www.hud.gov/i_want_to/talk_to_a_housing_counselor 
+          label: "Talk to a housing counselor"
       - subhead: Call us
```

Comment:
@bethpotts I had to resolve a conflict with `master` around here. Can you make sure it looks okay?

Reply:
@ncksllvn I'm asking Mikki. Sorry, just seeing this...

Reply:
@peggygannon is going to remove this "Talk to a housing counselor" link.